cinema/serializers.py

```diff
@@ -1 +1,105 @@
-# write serializers here
+from rest_framework import serializers
+from rest_framework.serializers import ModelSerializer
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+
+
+class GenreSerializer(ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
```

Comment:
you specified field names in all project except this place

Reply:
keep it one-styled

Reply:
I specified fields in most serializers because task required json objects with specific representation. I kept is as "__all__" for serializers which use only fields that are provide in models.